roles/openshift_logging/files/curator.yml

```diff
@@ -0,0 +1,18 @@
+# Logging example curator config file
```

Comment:
@jcantrill agree on all parts, I might have missed it, but is there a way to override the contents of the file included in the role when creating/updating the config map? If not, there should be. The role should be able to be consumed without modification of the files contained in the role itself.